pkg/runtime/config.go

```diff
@@ -317,6 +318,17 @@ func (c *Config) toInternal() (*internalConfig, error) {
 		outboundHealthz:           healthz.New(),
 	}
 
+	placementAddrs := strings.Split(strings.TrimPrefix(c.ActorsService, "placement:"), ",")
+	hasValidPlacementAddr := false
+	for _, addr := range placementAddrs {
+		if strings.TrimSpace(addr) != "" {
+			hasValidPlacementAddr = true
+			break
+		}
+	}
+
```

Comment:
Why wouldn’t a non valid placement address not error?

Reply:
The double negative is confusing me, but I did double check tho that this is no longer needed with the injector respecting the user annotations now for `""` && `" "`. removed the check